barcode/barcode_utils.py

```diff
@@ -49,9 +49,9 @@ def generate_barcode_csv(range_start, range_end):
 
 
 def generate_barcode_from_badge_num(badge_num, event_id=None, salt=None, key=None):
```

Comment:
This didn't exist at the time you wrote this plugin, but we can now say 

```python
SecretConfig.include_plugin_config(config['secret'])
```

and then you'll be able to say ``c.BARCODE_EVENT_ID``.